src/Linters/LintError.hack

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+/**
+ * A problem detected by a lint rule
+ *
+ * LintError is more generic than SingleRuleLintError because LintError is associated with a
+ * LintRule, not a SingleRuleLinter.
+ */
+<<__Sealed(
+  SingleRuleLintError::class,
+  // HHClientProblem::class
+)>>
+interface LintError {
+  public function getPath(): string;
+
+  public function getPosition(): ?(int, int);
```

Comment:
I don't feel strongly about this, but if it's going to be a BC-heavy release, maybe it's time to remove this for API simplification? it's equivalent for the first part of the getRange tuple, and the second part is nullable.

Reply:
Will do in another PR

Reply:
https://github.com/hhvm/hhast/pull/385